src/UI/Activity/Queue/TimeleftCell.js

```diff
@@ -10,24 +10,39 @@ module.exports = NzbDroneCell.extend({
         this.$el.empty();
 
         if (this.cellValue) {
-            if (this.cellValue.get('status').toLowerCase() === 'pending') {
-                var ect = this.cellValue.get('estimatedCompletionTime');
-                var time = '{0} at {1}'.format(FormatHelpers.relativeDate(ect), moment(ect).format(UiSettingsModel.time(true, false)));
+            var status = this.cellValue.get('status').toLowerCase();
+            var ect = this.cellValue.get('estimatedCompletionTime');
+            var time = '{0} at {1}'.format(FormatHelpers.relativeDate(ect), moment(ect).format(UiSettingsModel.time(true, false)));
+
+            if (status === 'pending') {
                 this.$el.html('<div title="Delaying download till {0}">-</div>'.format(time));
-                return this;
-            }
+            } else if (status === 'downloadclientunavailable') {
+                this.$el.html('<div title="Retrying download at {0}">-</div>'.format(time));
```

Comment:
Yeah, good call, fallback is being added and for the time being at least they will not be displayed in the UI (filtered when building the queue), I like the idea of being able to hide them, so some filtering would be in order, but I don't think it's required.